external/src/main/java/com/redhat/cloud/policies/engine/handlers/TriggersHandler.java

```diff
@@ -234,7 +241,7 @@ public void createFullTrigger(RoutingContext routing) {
                         trigger.setId(Trigger.generateId());
                     } else {
                         Trigger found = null;
-                        try {
+                        try (Scope ignored = tracer.buildSpan("getTrigger").asChildOf(serverSpan).startActive(true)) {
```

Comment:
``.startActive()`` is deprecated and discouraged.

Reply:
We are using opentracing v0.31 here, where this is valid.
The `tracer.activateSpan(span)` of OT.latest is not available.